tests/cloudservicesuploadadapter.js

```diff
@@ -39,53 +43,37 @@ describe( 'CloudServicesUploadAdapter', () => {
 	describe( 'init()', () => {
 		it( 'should set loader', () => {
 			UploadGatewayMock.lastToken = undefined;
+			TokenMock.initialToken = 'token';
 
 			return ClassicTestEditor
 				.create( div, {
 					plugins: [ CloudServicesUploadAdapter ],
 					cloudServices: {
-						token: 'abc',
+						tokenUrl: 'abc',
 						uploadUrl: 'http://upload.mock.url/'
 					}
 				} )
 				.then( editor => {
-					expect( UploadGatewayMock.lastToken ).to.equal( 'abc' );
+					expect( UploadGatewayMock.lastToken.value ).to.equal( 'token' );
```

Comment:
You should not check internals of the token class here. I would remove this assertion since this test is not about it.